packages/pn-personafisica-webapp/src/components/Contacts/DigitalContactElem.tsx

```diff
@@ -238,10 +251,16 @@ const DigitalContactElem = forwardRef<{ editContact: () => void }, Props>(
                   onClick={toggleEdit}
                   sx={{ mr: 2 }}
                   disabled={editDisabled}
+                  id="modifyContact"
```

Comment:
Be aware this id is not unique. You can use the concatenation between this string and f.id

Reply:
f.id not are here, is a variable of fields maps

Reply:
ok use the concatenation with senderId